### DISCUSSION TITLE
Discussion about the unreal engine plugin for xumm.

### DISCUSSION THREAD

**Discussion by Aaditya-T** [XRPLBounties/Proposals#123] [Ideas]
I had a few questions about this bounty, https://github.com/XRPLBounties/Proposals/blob/main/open_bounties/0014%20Unreal%20Engine%205%20Integration.md
1. Has anyone started work on this? (ig not, as there is no open pr)
2. Is there any preferred language for the plugin to be in? I see majority of the plugins are in c++, but am more comfortable with python.

**Comment by JST5000** [comment]
Hey @Aaditya-T, as far as I know no one has started work on this, and the plugin can be in whatever language you'd like :)

**Reply by Aaditya-T** [reply]
Great, i will do spme research and start working on it!
